tests/e2e/chains/ethereum/scripts/lodestar_entrypoint.sh

```diff
@@ -21,7 +21,7 @@ if [ -z "$genesisHash" ]; then
 	exit 1
 fi
 
-electra_fork_epoch=${EPOCH_ELECTRA}
+electra_fork_epoch=${EPOCH_PECTRA}
```

Comment:
nits: this `electra_fork_epoch` can be removed, and then replace `'${electra_fork_epoch}'/g'` with `${EPOCH_PECTRA}`

Reply:
I agree, but I wonder if Okubo-san had some intention on this point.

@toshihiko-okubo any comment?

Reply:
You can delete it, so it's okay to change it.

Reply:
Thank you everyone! I have fixed this: https://github.com/datachainlab/cosmos-ethereum-ibc-lcp/pull/74/commits/c9c6c0484ff1c44fe8c5a61c0bc1860aa3f3b87d